### PR TITLE
SBERDOMA-893 add new access 'canShareTickets'

### PR DESCRIPTION
**Pull request by leonid-d**

### CODE REVIEW

**Review by Dimitreee — APPROVED** [review]

File: apps/condo/pages/ticket/[id]/index.tsx [hunk 1]
```diff
@@ -208,7 +208,9 @@ const TicketIdPage = () => {
     }, () => { refetchComments() })
 
     const { link, organization } = useOrganization()
-
+    console.log(get(link, 'role'))
```
Comment (nomerdvadcatpyat) [inline apps/condo/pages/ticket/[id]/index.tsx:211]: remove log please

File: apps/condo/pages/ticket/[id]/index.tsx [hunk 2]
```diff
@@ -208,7 +208,9 @@ const TicketIdPage = () => {
...
+    const canShareTickets = get(link, 'role.canShareTickets')
+    console.log(canShareTickets)
```
Comment (nomerdvadcatpyat) [inline apps/condo/pages/ticket/[id]/index.tsx:213]: remove log please

File: apps/condo/pages/ticket/[id]/index.tsx (hunk 1 above)
Comment (pahaz) [inline apps/condo/pages/ticket/[id]/index.tsx:211]:
```suggestion
```

**Review by pahaz — APPROVED** [review]

**Comment by MrFoxPro** [comment]
Please make rebase and check migrations on test data before merging

**Pull request merged**